runtime/interpreter/encoding_test.go

```diff
@@ -358,6 +358,51 @@ func TestEncodeDecodeComposite(t *testing.T) {
 		testEncodeDecode(t,
 			encodeDecodeTest{
 				value: expected,
+				encoded: []byte{
```

Comment:
The diff is inferred incorrectly here, the actual change is duplicating the existing test and making it decode-only (to test backwards-compatibility), and adjusting the existing test to store the qualified identifier instead of the type identifier